dev/tools/test/testtcpdf.php

```diff
@@ -27,8 +27,8 @@
  * @since 2008-03-04
  */
 
-require_once '../../htdocs/includes/tecnickcom/tcpdf/config/tcpdf_config.php';
-require_once '../../htdocs/includes/tecnickcom/tcpdf/tcpdf.php';
+require_once DOL_DOCUMENT_ROOT.'/includes/tecnickcom/tcpdf/config/tcpdf_config.php';
```

Comment:
DOL_DOCUMENT_ROOT is not defined. This script is in a test dir to make test out of dolibarr so env is not loaded and DOL_DOCUMENT_ROOT is unknown.